src/oc_erchef/apps/oc_chef_wm/src/chef_wm_search.erl

```diff
@@ -360,7 +360,16 @@ make_query_from_params(Req) ->
 extract_path(_Item, []) ->
     null;
 extract_path(Item, Path) ->
-    ej:get(list_to_tuple(Path), Item, null).
+    try ej:get(list_to_tuple(Path), Item, null) of
+        Result -> Result
+    catch
+        %% Don't throw a 500 when the user input is invalid
+        %%   This typically happens when the user provides an invalid attribute
+        %%   key, typically trying to access a value as a hash that is actually
+        %%   a string (i.e. uptime.seconds)
+        error:function_clause -> null;
+        Error:Reason -> {Error, Reason}
```

Comment:
If the term that comes back can't be converted as ejson -> json, we'll still end up with a 500 as we prepare the body response.  Maybe a generic error string for the return value in that case, and use `error_logger`to capture the details.